Validation/Geometry/test/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.root
```

Comment:
May I ask you why? IIUC correctly, this gitignore will  only be valid from this folder down, and we have no .root file in the release in this folder, nor we have plans to have root files committed in our source code repository, de we?